blog/zh/blog/2022/12/27/apisix-ingress-with-gatewayapi.md

```diff
@@ -0,0 +1,151 @@
+---
+title: "APISIX Ingress 对 Gateway API 的支持和应用"
+author: "lingsamuel"
+authorURL: "https://github.com/lingsamuel"
+authorImageURL: "https://github.com/lingsamuel.png"
+keywords: 
+- Apache APISIX
+- Ingress
+- Ingress controller
+- Gateway API
+description: 本文介绍了 Gateway API 这个将服务暴露到集群之外的全新规范，并且介绍了如何在 APISIX Ingress Controller 中使用它。
+tags: [Ecosystem]
+---
+
+> 本文介绍了 Gateway API 这个将服务暴露到集群之外的全新规范，并且介绍了如何在 APISIX Ingress Controller 中使用它。
+
+<!--truncate-->
+
+> 作者@lingsamuel，API7.ai 云原生技术专家，Apache APISIX Committer。
+
+Gateway API 是除原生 Service 与 Ingress 之外，APISIX 社区发起的独立规范，帮助用户将 Kubernetes 中的服务暴露到集群之外，由  [sig-network](https://github.com/kubernetes/community/tree/master/sig-network) （特别兴趣小组）管理。
+
+Gateway API 支持大部分常用网络协议（HTTP、TCP、UDP 等）和对 TLS 的支持。此外，Gateway API 中的 Gateway 资源能够通过 Kubernetes API 来管理代理和网关的生命周期。
+
+随着 Gateway API 被[广泛实现](https://gateway-api.sigs.k8s.io/implementations/)与应用，Gateway API 发布了 v0.5.0 版本。在该版本中，一些核心 API 首次进入 Beta 阶段，包括：GatewayClass、Gateway、HTTPRoute。
+
+### **Gateway API 优势盘点**
```

Comment:
You can remove **, because the font of the title itself will be bold, and there is no need to set it separately here.

The title of this article needs to be updated